examples/io_perf_test.cpp

```diff
@@ -36,12 +36,35 @@ print_elapsed_time(
     std::cout << message << ": " << dur/1000000.0 << " [s]" << std::endl;
 }
 
+void
+print_version_map()
+{
+    std::cerr << "current version map: "  << std::endl;
+    auto fammap = otio::family_label_version_map();
+    for (auto kv_fam : fammap)
+    {
+        std::cerr << kv_fam.first << std::endl;
+        for (auto kv_lbl: kv_fam.second)
+        {
+            std::cerr << "  " << kv_lbl.first << std::endl;
+            for (auto kv_schema_version : kv_lbl.second)
+            {
+                std::cerr << "    \"" << kv_schema_version.first << "\": ";
+                std::cerr << kv_schema_version.second << std::endl;
```

Comment:
I assume there's no performance constraint here? i.e. print_version_map doesn't have to be fast? because std::endln flushes io in addition to inserting a "\n" in the stream.

Reply:
io_perf_test is just an example file - I have it dumping the core schemas in its startup to stdout just for info/debugging.  I can remove that function if you think its a distraction.  It was more useful when there was code allowing the family/labels to be set in the core library.